### PR TITLE
Added more meta description

### LINKED ISSUE #364: Meta Description & Open Graph
What a spectacularly lame embed!
slack_2016-02-17_23-47-04

Would be awesome if someone with HTML skills and some good words could fix this.
- [x] Add a `<meta name="description" .../>` with good words

Bonus:
- [x] Add [Open Graph markup](https://developers.facebook.com/docs/sharing/webmasters#markup) with good words/URLs

### PR DESCRIPTION
**Pull request by hobbyexplorer**
Fixes #364

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <head>
   <meta charset='utf-8' />
   <meta http-equiv="X-UA-Compatible" content="chrome=1" />
-  <meta name="description" content="Up For Grabs : Jump in!" />
+  <meta property="og:url"         content="http://up-for-grabs.net" />
+  <meta property="og:type"        content="product" />
+  <meta property="og:title"       content="Up For Grabs" />
+  <meta property="og:description" content="Want to contribute to open source, but github feels like an ocean? Grab these tiny issues before they dissolve themselves!" />
```
Comment (shiftkey) [inline _includes/head.html]: I'm gonna defer to @dahlbyk on the words here, but please make it `GitHub` at least here

Comment (hobbyexplorer) [inline _includes/head.html]: Sure @shiftkey, pushed the change.

File: _includes/head.html [hunk 2]
```diff
@@ -1,7 +1,11 @@
 <head>
   <meta charset='utf-8' />
   <meta http-equiv="X-UA-Compatible" content="chrome=1" />
-  <meta name="description" content="Up For Grabs : Jump in!" />
```
Comment (shiftkey) [inline _includes/head.html:4]: Should we keep this tag around (with some more awesome words) for those browsers who don't care for the Open Graph spec?

File: _includes/head.html [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
+  <meta property="og:url"         content="http://up-for-grabs.net" />
+  <meta property="og:type"        content="product" />
```
Comment (dahlbyk) [inline _includes/head.html]: I don't think we're a `product`. I'd stick with `website`, or drop this altogether.

> If you don't specify a type,the default is `website`.

File: _includes/head.html (hunk 2 above)
Comment (dahlbyk) [inline _includes/head.html:4]: :+1: Aligning with `og:description` here makes sense, assuming that value isn't more than [~155 characters](https://moz.com/learn/seo/meta-description).

File: _includes/head.html [hunk 4]
```diff
@@ -1,7 +1,11 @@
...
+  <meta property="og:title"       content="Up For Grabs" />
+  <meta property="og:description" content="Want to contribute to open source, but GitHub feels like an ocean? Grab these tiny issues before they dissolve themselves!" />
```
Comment (dahlbyk) [inline _includes/head.html]: We do have a few non-GitHub projects... "Want to contribute to open source, but not sure where to start?"

I'm also not sure exactly what "before they dissolve themselves" means?

Comment (hobbyexplorer) [inline _includes/head.html]: Hi @dahlbyk 
Added the edit now
Thanks!

File: _includes/head.html (hunk 3 above)
Comment (hobbyexplorer) [inline _includes/head.html]: Hi @dahlbyk 
Sure, I removed that line altogether.
Thanks!

File: _includes/head.html (hunk 2 above)
Comment (hobbyexplorer) [inline _includes/head.html:4]: Hi @shiftkey  
I aligned the description tag with `og:description` now
Thanks!

**Comment by hobbyexplorer** [comment]
Tested the modifications using `ngrok` and [Facebook URL debugger](https://developers.facebook.com/tools/debug/), attached screen cap below
screen shot 2016-02-19 at 11 05 15 pm

**Comment by shiftkey** [comment]
This all looks good - thanks for the excellent work @magevadi!



I'm gonna merge this in and then leave it to @dahlbyk to shout at me if I missed something :grin:

**Pull request merged**